answers.js

```diff
@@ -0,0 +1,107 @@
+"use strict";
+
+/*
+1. Who is bigger?
+*/
+function random(number1, number2) {
```

Comment:
Although a bit less efficient, I would use Math.min and Math.max here instead of this if:

```
return between(Math.min(n1, n2), Math.max(n1, n2))
```